### DISCUSSION TITLE
kale calculator

### DISCUSSION THREAD

**Discussion by daryledesilva** [Kale-Network/kale-blockchain#24] [General]
Hi, this is the dev for chiaforkscalculator.com. Listed Kale here: https://chiaforkscalculator.com/
and the dedicated calculator for Kale can be found here: https://chiaforkscalculator.com/kale
if you have any questions/comments/suggest feel free to reach out and let me know :) thank you!

**Comment by kaleseed** [comment]
@daryledesilva great work! We have put the calculator on the main page of our website. Thanks.

**Reply by daryledesilva** [reply]
thanks a lot @kaleseed ! let me know if  you need anything related to the calculator site. 😁
